src/jsMain/js/FakeClientDevice.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import Draggable from 'react-draggable';
+import styles from './FakeClientDevice.scss';
+
+export default class FakeClientDevice extends Component {
+  constructor(props) {
+    super(props);
+    this.width = props.width / 2;
+    this.height = props.height / 2;
+    this.onClose = props.onClose;
+    this.onResize = props.onResize;
+  }
+
+  onZoomOut = () => {};
+  onZoomIn = () => {};
+  onDragStart = (e, draggableData) => {
+    const draggableNode = draggableData.node;
+    const contentNode = draggableNode.getElementsByClassName(styles['FakeClientDevice--content'])[0];
+    let eventNode = e.target;
+    while (eventNode != null) {
+      if (eventNode === contentNode) {
+        console.log("in content, false");
+        return false;
+      } else if (eventNode === draggableNode) {
+        console.log("in draggable, true");
+        return true;
+      } else {
+        eventNode = eventNode.parentNode;
+      }
+    }
+  };
+
+  render() {
+    const borderWidth = 10;
+    const containerStyle = {
+      width: (this.width + borderWidth * 2) + 'px',
+      height: (this.height + borderWidth * 2) + 'px',
+    };
+    const contentStyle = {
+      width: this.width + 'px',
+      height: this.height + 'px',
+    };
+
+    return <Draggable onStart={(e, data) => { return this.onDragStart(e, data); }}>
+      <div className={styles['FakeClientDevice--pad']} style={containerStyle}>
+        <div className={styles['FakeClientDevice-controls']}>
+          <i className="fas fa-search-minus" onClick={_ => this.onZoomOut()}/>
```

Comment:
For anonymous functions with no parameters, we could just write this as:
```
() => this.onZoomOut()
```

instead of
```
_ => this.onZoomOut()
```